tensorflow_probability/python/bijectors/real_nvp_test.py

```diff
@@ -97,6 +97,46 @@ def testBatchedBijectorWithMLPTransform(self):
       self.assertAllClose(x_, inverse_y_, rtol=1e-1, atol=0.)
       self.assertAllClose(ildj_, -fldj_, rtol=1e-6, atol=0.)
 
+  def testBijectorConditionKwargs(self):
+    N = 3
+    x_ = np.arange(N * 4 * 2).astype(np.float32).reshape(N, 4 * 2)
+    conditions = {
+        'a': tf.reshape(tf.range(N * 4, dtype=tf.float32), (N, 4)),
+        'b': tf.reshape(tf.range(N * 2, dtype=tf.float32), (N, 2)),
+    }
+
+    with self.test_session() as sess:
```

Comment:
While the rest of this file uses this style, I'm trying to move the codebase to a newer style (which should hopefully make it easier for you as well!)

You can usually prefer self.evaluate(...) instead of sess.run(...). This allows you to also remove this self.test_session scope (and the tf.global_variables_initializer can be self.evaluate(tf.global_variables_initializer())

self.evaluate(...) underneath the hood uses/reuses a tf.Session().

Reply:
Nice, I did not know about `self.evaluate`. Thanks!